project.clj

```diff
@@ -54,15 +54,11 @@
                    :sql-store :sql-store
                    :es-store :es-store
                    :disabled :disabled
-                   :es-producer #{:es-producer
-                                   :es-producer-filtered-alias
-                                   :es-producer-aliased-index}
                    :default #(not= :disabled %)
-                   :integration #{:es-store
-                                   :integration
-                                   :es-producer
-                                   :es-producer-filtered-alias
-                                   :es-producer-aliased-index}
```

Comment:
this selector syntax was not working, using `lein test :integration` launched 0 tests so I reverted back to use a fn instead. tell me if I got it wrong.